requirements.txt

```diff
@@ -1,189 +1,36 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
It says this was pip-compile'd with Python 3.10, but the CI test matrix only tests against Python 3.8/3.9: https://github.com/ThreeSixtyGiving/360insights/pull/208/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR17

We probably do want to support 3.10, so perhaps add 3.10 to the CI test matrix?
Or are the python tests even relevant in this way since most of the backend is gone?

I guess we should focus on the Python version used in production/deployment - in the dokku buildpack/container?